### ISSUE TITLE
Steamdeck desktop no way to reset theme

### ISSUE THREAD

**Issue by lifeiscontent** [ValveSoftware/SteamOS#1485]
#### Your system information

* Steam client version:
* SteamOS version: 3.5.19
* Opted into Steam client beta?: [Yes/No] No
* Opted into SteamOS beta?: [Yes/No] No
* Have you checked for updates in Settings > System?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen.

#### Steps for reproducing this issue:

1. go to global theme
2. see that defaults apply appearance settings, but not window settings
3. see theme update, but steam deck menu icon is lost

**Comment by manuelafm** [comment]
I changed themes, to all of the 6 available by default, and didn't see anything strange.  I assume that with "steam start menu icon" you mean the one in the bottom-left corner in the panel, the one that opens the main menu.

The icon changes slightly among themes, for some the outer part (half-circumference) is white, with other it is black.

**Comment by lifeiscontent** [comment]
@manuelafm yeah, I'm not sure how, but between the default themes I've managed to completely break the UI customizations by fully resetting to the I believe the theme was called breeze? its one of the defaults that comes with the OS.

**Comment by bertogg** [comment]
This sounds like a bug in the Plasma desktop. If you manage to reproduce the problem again and know the exact steps we could try to narrow it down. Or you can try to see in a KDE users forum if anyone had the same problem, as I said this sounds generic and not specific to SteamOS.

Anyway, I think that the theme configuration is in `/home/deck/.config/kdeglobals` and `/home/deck/.config/kdedefaults`, if you feel confident you can have a look there and see if there is anything suspicious.

**Comment by lifeiscontent** [comment]
@bertogg cool, when I get some downtime to goof around with it again, I'll report back.

**Comment by lifeiscontent** [comment]
@bertogg this is still an issue, if I re-image my OS and select these options:

![image](https://github.com/ValveSoftware/SteamOS/assets/180963/c62e8aca-feed-4b05-aff1-a6731d3233d2)

there's no way to get the default settings back because "Vapor" theme doesn't modify window layout which it should but doesn't.

now whenever I try to "reset" the default, it doesn't update and the steam icon is lost for the start menu icon

**Comment by davidedmundson** [comment]
It seems this is a bug on the KDE side, we shall follow up.
